mmv1/third_party/terraform/utils/iam.go.erb

```diff
@@ -448,6 +448,63 @@ func compareBindings(a, b []*cloudresourcemanager.Binding) bool {
 	return reflect.DeepEqual(aMap, bMap)
 }
 
+// Returns a map representing iam bindings that are in one map but not the other.
```

Comment:
These functions might go better in another file but I'm not sure which one. Their main purpose is to provide more information about two sets of bindings than `compareBindings` does.

Reply:
My inclination would be to but this in bootstrap_utils_test for now because that's the only code using it - but I don't feel super strongly about it.

I do think it could make sense to rename the functions to something like `bindingsDiff` or `diffBindings` to indicate that it returns the diff instead of just an equality check. (It's not a true diff since it doesn't let you know which were missing in the actual vs expected, but I think that would probably still be less confusing.)

Reply:
I was planning to use this to generate a better error message on line 977 of resource_composer_environment_test.go.erb, but I'll do that in a followup PR.